libs/wingsdk/src/std/test.ts

```diff
@@ -38,8 +38,12 @@ export class Test extends Resource implements IInflightHost {
     return App.of(scope).newAbstract(TEST_FQN, scope, id, inflight, props);
   }
 
-  /** @internal */
-  public readonly _fn: Function;
+  /**
+   * The function that will be called when the test is run. This will not be created
+   * if the app is not compiled as a test.
```

Comment:
Comment is not entirely correct (will also be created for sim). Also the double negative could probably be avoided.

Reply:
I also don't not dislike double negatives 😄